packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/health-monitor/edit-name/controller.js

```diff
@@ -0,0 +1,52 @@
+export default class OctaviaLoadBalancerHealthMonitorEditNameCtrl {
+  /* @ngInject */
+  constructor(OctaviaLoadBalancerHealthMonitorService, Alerter, $translate) {
+    this.isLoading = false;
+    this.OctaviaLoadBalancerHealthMonitorService = OctaviaLoadBalancerHealthMonitorService;
+    this.Alerter = Alerter;
+    this.$translate = $translate;
+  }
+
+  $onInit() {
+    this.name = this.healthMonitor.name;
```

Comment:
Why create a new var instead of directly using helathMonitor.name ?

Reply:
Doing like this allow me to not trigger the parent view change when I update the name